### PR TITLE
Add TAI & TAP for xcm

### PR DESCRIPTION
**Pull request by zjb0807**

### CODE REVIEW

**Review by xlc — DISMISSED** [review]

**Review by shaunxw — DISMISSED** [review]

**Comment by zqhxuyuan** [comment]
We need add trader config item too, or do we want it be registered as ForeignAsset?

**Comment by xlc** [comment]
Yeah let's add to trader config for now. Later once those tokens are listed on swap then we can enable tx fee pool.

**Comment by zjb0807** [comment]
`aUSD:DOT = 40:1`
`kUSD:KSM = 400:1`
should we adjust the `Trader` fee?

**Comment by zqhxuyuan** [comment]
> `aUSD:DOT = 40:1` `kUSD:KSM = 400:1` should we adjust the `Trader` fee?

DOT/KSM/aUSD is prior processed by `BuyWeightRateOfTransactionFeePool`, the config here is used for testcase mostly.

**Review by zqhxuyuan — APPROVED** [review]

File: runtime/acala/src/xcm_config.rs [hunk 1]
```diff
@@ -135,6 +135,15 @@ parameter_types! {
 		).into(),
 		aca_per_second()
 	);
+	pub TapPerSecond: (AssetId, u128) = (
+		MultiLocation::new(
+			0,
+			X1(GeneralKey(TAP.encode())),
+		).into(),
+		// TODO: No price yet, assumed set at 4340
```
Comment (xlc) [inline runtime/acala/src/xcm_config.rs:143]: add an issue to update this later

**Review by xlc — APPROVED** [review]

**Pull request merged**